modules/ROOT/pages/otel-support.adoc

```diff
@@ -0,0 +1,145 @@
+= OpenTelemetry Support in Mule Runtime
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: otel, otel support, OpenTelemetry
+
+OpenTelemetry is an observability standard consisting of Specifications, APIs, and SDKs. It helps instrument, generate, collect, and export telemetry data, such as metrics, logs, and traces, to analyze software behavior. 
+
+OpenTelemetry enables Mule runtime to provide observability into the behavior of Mule apps. With distributed tracing, Mule runtime also brings observability to the interactions between different Mule apps and non-Mule components when the latter leverages this standard.
```

Comment:
I think you can say "Mule runtime engine" for the first mention and then just use "Mule" for subsequent references.